### PR TITLE
Update dependencies and set up automatic build

### PR DESCRIPTION
**Pull request by vetler**
Updating to Java 8 and newer dependencies. Also setting up automatic build with GitHub  actions.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -47,10 +47,10 @@
     </distributionManagement>
 
     <properties>
-        <java.version>1.7</java.version>
-        <logback.version>1.2.0</logback.version>
-        <junit.version>4.13.1</junit.version>
-        <jackson-databind.version>2.10.0.pr1</jackson-databind.version>
+        <java.version>1.8</java.version>
```
Comment (maricn) [inline pom.xml]: doesn't seem that any of the updated dependencies would require java 8 upgrade (jackson-databind says java 7 works for it until `2.12`, and the rest are just revision updates).. 

do we really need to drop support for java 7 just for these deps updates? i'm fine with updating the deps further (their major versions), but seems like we wouldn't capitalize over such change in this PR..

Comment (vetler) [reply pom.xml]: No, we can absolutely keep Java 7, seems like the best thing to do.
Fixed!

**Review by maricn — APPROVED** [review]

**Review by maricn — COMMENTED** [review]
thank you for submitting the PR, feel free to merge, and i'll see into releasing it these days.. (i'm not sure I have access to release keys on the laptop that i have with me on my vacation)

**Comment by maricn** [comment]
@vetler release is out, please use the latest 1.6.1 version

**Pull request merged**